schema/me/conversation/index.js

```diff
@@ -167,12 +167,36 @@ export const ConversationFields = {
   },
   last_message: {
     type: new GraphQLNonNull(GraphQLString),
+    description: "This is a snippet of text from the last message.",
     resolve: conversation => {
       return get(conversation, "_embedded.last_message.snippet")
     },
   },
   last_message_at: date,
 
+  // This is our desired API for an unread indicator, for a given email:
+  // If the given email address is a recipient of the last message, return true.
+  is_last_message_to: {
+    type: GraphQLBoolean,
+    args: {
+      email_address: {
+        type: new GraphQLNonNull(GraphQLString),
+      },
+    },
+    resolve: (conversation, { email_address }) => false,
```

Comment:
Next iteration can be the `last_message.from_email_address !== email_address`.

This basically assumes that if the user didn't send the message....then the user was a recipient. That led to the bug where messages in a conversation between two partner contacts will still show up in the inbox (but not be delivered via email) to the user.